workers/unity/Assets/Playground/Scripts/Player/ProcessLaunchCommandSystem.cs

```diff
@@ -18,8 +18,7 @@ private struct LaunchCommandData
             [ReadOnly] public EntityArray Entity;
             public ComponentDataArray<SpatialOSLauncher> Launcher;
 
-            [ReadOnly]
-            public ComponentArray<CommandRequests<Generated.Playground.Launcher.LaunchEntity.Request>> CommandRequests;
+            [ReadOnly] public ComponentArray<CommandRequests<Generated.Playground.Launcher.LaunchEntity.Request>> CommandRequests;
```

Comment:
I've had linting fail this yesterday in the cloud FYI (although it doesn't seem like it does all the time)

Reply:
I mean these are the changes made when I ran ci/lint.sh so it shouldn't fail. *He says wishfully*

Reply:
Yeah its not consistent at all, its really sad